versions/next/asyncapi.md

```diff
@@ -1405,40 +1406,39 @@ The following properties are taken directly from the JSON Schema definition and
 - uniqueItems
 - maxProperties
 - minProperties
-- required
 - enum
+- const
+- examples
+- if / then / else
+- readOnly
+- writeOnly
 
 The following properties are taken from the JSON Schema definition but their definitions were adjusted to the AsyncAPI Specification.
 
-- type - Value MUST be a string. Multiple types via an array are not supported.
+- properties - Inline or referenced schema MUST be of a [Schema Object](#schemaObject) and not a standard JSON Schema.
+- patternProperties - Inline or referenced schema MUST be of a [Schema Object](#schemaObject) and not a standard JSON Schema.
+- additionalProperties - Inline or referenced schema MUST be of a [Schema Object](#schemaObject) and not a standard JSON Schema.
+- additionalItems - Inline or referenced schema MUST be of a [Schema Object](#schemaObject) and not a standard JSON Schema.
+- items - Inline or referenced schema MUST be of a [Schema Object](#schemaObject) and not a standard JSON Schema.
+- propertyNames - Inline or referenced schema MUST be of a [Schema Object](#schemaObject) and not a standard JSON Schema.
+- contains - Inline or referenced schema MUST be of a [Schema Object](#schemaObject) and not a standard JSON Schema.
```

Comment:
should `not` be on here?

Reply:
It's below. It was already there, that's why it's not showing the change.